sapl/templates/materia/materialegislativa_filter.html

```diff
@@ -139,6 +139,16 @@ <h3>{% trans 'Pesquisa concluída com sucesso! Foi encontrada 1 matéria.'%}</h3
                 {% if m.em_tramitacao %}
                   <a href="{% url 'sapl.materia:acompanhar_materia' m.id %}">Acompanhar Matéria</a>
                 {% endif %}
```

Comment:
Acho que para manter o padrão da tela é melhor deixar o link de acompanhamento embaixo dos resultados de audiência pública. Também é importante colocar uma quebra de linha para melhor visualização. Não acha?
Deixar algo como no exemplo abaixo:
![image](https://user-images.githubusercontent.com/19825887/47719190-2659f680-dc29-11e8-9890-a344f0c58380.png)

Minha sugestão é o código ser algo assim:
```suggestion
                {% if m.audienciapublica_set.exists %}
                  <strong>Audiência(s) Pública(s): </strong>
                  {% for ap in m.audienciapublica_set.all %}
                    <a href="{% url 'sapl.audiencia:audienciapublica_detail' ap.id %}">
                        {{ ap.nome }}
                    </a>
                    </br>
                    {% if not forloop.last %},&nbsp;{% endif %}
                  {% endfor %}
                {% endif %}
                <p></p>
                {% if m.em_tramitacao %}
                  <a href="{% url 'sapl.materia:acompanhar_materia' m.id %}">Acompanhar Matéria</a>
                {% endif %}
```

Reply:
Corrigida a ordem dos itens. A quebra de linha porém, acho que não é necessária entre as audiências pois elas estão separadas por vírgula na mesma linha.
![screenshot from 2018-10-30 15-33-20](https://user-images.githubusercontent.com/20192488/47741504-37baf700-dc59-11e8-8f9a-24c025fea39d.png)